R/match.R

```diff
@@ -108,9 +108,43 @@
 #'     `"last"` if you just need to detect if there is at least one match.
 #'   - `"first"` returns the first match detected in `haystack`.
 #'   - `"last"` returns the last match detected in `haystack`.
-#'   - `"warning"` throws a warning if multiple matches are detected, but
-#'     otherwise falls back to `"all"`.
-#'   - `"error"` throws an error if multiple matches are detected.
```

Comment:
Removed from the docs. No formal deprecation with `deprecate_soft()` yet, I'd like to remove usage of them from dplyr and ivs first